_posts/2021-09-18-p2pool-released.md

```diff
@@ -0,0 +1,49 @@
+
+All pool blocks pay out to miners immediately. Because there is no pool wallet, funds are never in custody --- It's trustless.
+
+P2Pool uses PPLNS payout scheme, has 0% fee and 0 XMR payout fee (by design, fees can't be added). Everyone in the PPLNS window is rewarded directly via the coinbase transaction reward. Minimum payout is less than 0.0005 XMR due to the very efficient miner reward transaction format.  --- each individual miner payout takes only 38 bytes on the Monero blockchain!
+
+### Why Monero P2Pool?
+
+There's been some worry about mining pool centralization and the possibility of 51% attacks. Since atomic swaps went live, solving exchange delisting/KYC threat, mining is now the most centralized part of the Monero ecosystem. Currently, one pool accounts for almost 40% of Monero's network hashrate; this is not a stable situation long-term, because a pool admin can be pressured to do malicious things, shutdown their pool, or expropriate funds without paying miners.
+
+### How does it work?
+
+P2Pool is a sidechain to Monero and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them. If a block is good enough to be a Monero block it is also submitted to the Monero network to be included in its blockchain. Once the Monero block is confirmed by the network, those miners are directly paid in the coinbase because they were included already in the block. If P2Pool "shares" are found a the same block height as an existing share, it is still included as an uncle block (worth 20% less than a normal share) so you can still get paid for it. (Uncle blocks can be submitted up to 3 blocks behind the current height and still be included.)
+
+### Features
+
+- Decentralized: no central server that can be shutdown/blocked.
+- Permissionless: there is no one to decide who can mine on the pool and who can't.
+- Trustless: there is no pool wallet, funds are never in custody. All pool blocks pay out to miners immediately.
+- PPLNS payout scheme
+- 0% fee and no payout fee
+- Less than 0.0004 XMR minimum payout
+- Fast block times, down to 1 second
+- Uncle blocks are supported to avoid orphans - all your shares will be accounted for!
+- Advanced mempool picking algorithm, it creates blocks with better reward than what monerod solo mining does
+- Password protected private pools
+- Configurable PPLNS window size and block time
+
+*NOTE: Setting a password or changing any settings in config.json will create a separate sidechain. Only do this if you know what you're doing.*
+
+### Getting Started
+
+- [P2Pool Download](https://github.com/SChernykh/p2pool/releases)
+- [P2Pool GitHub](https://github.com/SChernykh/p2pool)
+- P2Pool IRC channels: #monero-pow and #p2pool-log on [Libera.Chat](https://libera.chat/).

```

Comment:
Maybe add a link to https://p2pool.io/ here? WIth a note that it's not a central pool server, just a monitoring page.

Reply:
Also, since it's a 0% fee pool, having my donation address in the article would be appreciated: 44MnN1f3Eto8DZYUWuE5XZNUtE3vcRzt2j6PzqWpPau34e6Cf4fAxt6X2MBmrm6F9YMEiMNjN6W4Shn4pLcfNAja621jwyg

Reply:
Ack. Added and added.